big-a/src/_nav.js

```diff
@@ -17,16 +17,6 @@ import {
 import { CNavGroup, CNavItem, CNavTitle } from '@coreui/react'
 
 const _nav = [
-  {
-    component: CNavItem,
-    name: 'Dashboard',
-    to: '/dashboard',
```

Comment:
dashboard is removed but landing page remains at /dashboard. We can go about this two ways:
Remove /dashboard path from "routes.js" and update default landing page to be '/playbook'
or
Restore dashboard component and adjust the view of dashboard to reflect what we want.